jerry-core/CMakeLists.txt

```diff
@@ -194,6 +196,12 @@ if(FEATURE_SNAPSHOT_SAVE)
   set(DEFINES_JERRY ${DEFINES_JERRY} JERRY_ENABLE_SNAPSHOT_SAVE)
 endif()
 
+# Enable system allocator
+if(FEATURE_SYSTEM_ALLOCATOR)
+  set(DEFINES_JERRY ${DEFINES_JERRY} JERRY_SYSTEM_ALLOCATOR)
+  set(DEFINES_JERRY ${DEFINES_JERRY} JERRY_CPOINTER_32_BIT)
```

Comment:
Instead of sneakily enabling `JERRY_CPOINTER_32_BIT` here, I'd suggest setting `FEATURE_CPOINTER_32_BIT` somewhere before line 35, this way the status message will correctly print if the feature is enabled or not.